CHANGELOG.md

```diff
@@ -27,7 +27,7 @@
 - [v1.0.0](#v100)
 
 ## Unreleased
-
+- **General:** Using manager client in  `keda-metrics-apiserver` to avoid flush request to apiserver([2914](https://github.com/kedacore/keda/issues/2914))
```

Comment:
please move this to `Improvements` section below.

Reply:
also please change the message to this please:
```
- **General:** Using manager client in KEDA Metrics Server to avoid flush request to Kubernetes Apiserver([2914](https://github.com/kedacore/keda/issues/2914))
```

Reply:
updated